.github/pythonchecks.txt

```diff
@@ -18,3 +18,4 @@ mypy==0.931
 types-requests==2.27.7
 pytest-xdist==2.5.0
 requests==2.27.1
```

Comment:
This doesn't exist in Debian. We'll need to somehow install it on the appservers via puppet.